src/Umbraco.Core/Persistence/Repositories/ContentTypeBaseRepository.cs

```diff
@@ -1250,5 +1250,19 @@ public string GetUniqueAlias(string alias)
             while (aliases.Contains(test = alias + i)) i++;
             return test;
         }
+
+        /// <summary>
+        /// Given the path of a content item, this will return true if the content item exists underneath a list view content item
+        /// </summary>
+        /// <param name="contentPath"></param>
+        /// <returns></returns>
+        public bool HasContainerInPath(string contentPath)
+        {
+            var ids = contentPath.Split(',').Select(int.Parse);
+            var sql = new Sql(@"SELECT COUNT(pk) FROM cmsContentType

```

Comment:
This SQL statement doesn't work, throws an error: `Ambiguous column name 'pk'.` i'll fix up